public/javascripts/list.js

```diff
@@ -1,5 +1,5 @@
 const navBar = document.querySelector(".navBar");
-const search = document.querySelector('.search');
+// const search = document.querySelector('.search');
```

Comment:
why is this commented out?

Reply:
its the search option on the activeList I want to come back to that later